db/dataset_eval.py

```diff
@@ -113,7 +113,7 @@ def set_job_result(job_id, result):
     commit()
 
 
-def set_job_status(job_id, status):
+def set_job_status(job_id, status, status_msg=None):
```

Comment:
Yep, it's optional. I added docstring for this function in c7b5eedc69c2ffcb7bcf6d26608bf2f38dff220f. :scroll: